LeaderboardBackend/Controllers/RunsController.cs

```diff
@@ -1,49 +1,44 @@
-using Microsoft.AspNetCore.Mvc;
-using LeaderboardBackend.Models.Entities;
-using LeaderboardBackend.Services;
+using Microsoft.AspNetCore.Mvc;
+using LeaderboardBackend.Models.Entities;
+using LeaderboardBackend.Services;
 using Microsoft.AspNetCore.Http;
 using LeaderboardBackend.Controllers.Annotations;
 using LeaderboardBackend.Models.Requests;
 
-namespace LeaderboardBackend.Controllers
-{
-	[Route("api/[controller]")]
-	[ApiController]
-	public class RunsController : ControllerBase
-	{
-		private readonly IRunService _runService;
+namespace LeaderboardBackend.Controllers
+{
+	[Route("api/[controller]")]
+	[ApiController]
+	public class RunsController : ControllerBase
+	{
+		private readonly IRunService _runService;
+
+		public RunsController(IRunService runService)
+		{
+			_runService = runService;
+		}
 
-		public RunsController(IRunService runService)
-		{
-			_runService = runService;
-		}
-
 		/// <summary>Gets a Run.</summary>
 		/// <param name="id">The Run ID. Must parse to a long for this route to be hit.</param>
 		/// <response code="200">The Run with the provided ID.</response>
 		/// <response code="404">If no Run is found with the provided ID.</response>
 		[ApiConventionMethod(typeof(Conventions),
 							 nameof(Conventions.Get))]
-		[ProducesResponseType(StatusCodes.Status200OK)]
-		[ProducesResponseType(StatusCodes.Status404NotFound)]
 		[HttpGet("{id}")]
-		public async Task<ActionResult<Run>> GetRun(Guid id)
-		{
-			Run? run = await _runService.GetRun(id);
-			if (run == null)
-			{
-				return NotFound();
-			}
-
-			return Ok(run);
+		public async Task<ActionResult<Run>> GetRun(Guid id)
```

Comment:
Why did these change? I wish GitHub let you check if the change is because of spaces vs tabs.

Reply:
From what I can tell the file's still using tabs. I can change the values in the .editorconfig and it will take effect on save in VSCode, which makes me think I've got it set up right.

Maybe there's something other than the editorconfig contributing to the formatting. I'll look into it.

Reply:
If that's the case then it's reasonable to assume GitHub is being pissy I think. I don't actually see them doing anything specifically wrong so if something is weird we can fix it later.